modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/claim/management/v1/model/LocalClaimRes.java

```diff
@@ -209,6 +210,18 @@ public void setSupportedByDefault(Boolean supportedByDefault) {
         this.supportedByDefault = supportedByDefault;
     }
 
+    /**
+     * Specifies if the claim can hold multiple values.
+     **/
+    @ApiModelProperty(value = "Specifies if the claim can hold multiple values.")
+    @JsonProperty("multiValued")
+    public Boolean getMultiValued() {
+        return multiValued;
+    }
+    public void setMultiValued(Boolean multiValued) {
```

Comment:
```suggestion

    public void setMultiValued(Boolean multiValued) {
```

Reply:
Fixed by - https://github.com/wso2/product-is/pull/23619/commits/2e4dd291689b3e9fd029f966b7095f03f49c5ff5

Reply:
Fixed by - https://github.com/wso2/product-is/pull/23619/commits/c0196674373dfa6ad281ac00dca19336d752abdb